### DISCUSSION TITLE
there are no subfolders in the /var/azuracast directory

### DISCUSSION THREAD

**Discussion by jill88** [AzuraCast/AzuraCast#7644] [Q&A]
I'm trying to manage backups and free up space.  I went to the /var/azuracast folder and there are no subfolders like backups in there.  Where are all these folders?  I am using the normal docker install, so maybe they are inside the docker container only, I'm assuming?  If so, I'm going to setup an external location for all of this.  Thanks.

**Comment by DJMic** [comment]
Into /var/azuracast type `./docker.sh bash` and you'll be able to navigate to these subfolders

**Comment by LordHelmchen666** [comment]
Docker volumes can be found in `/var/lib/docker/volumes/`
AzuraCast automated backups in `/var/lib/docker/volumes/azuracast_backups/_data/`
Station data in `/var/lib/docker/volumes/azuracast_station_data/_data/`

I use cron job running once a day to copy on another server 
`rsync -avhe ssh --delete --progress --stats user@IP:/var/lib/docker/volumes/azuracast_backups/_data/ /home/backups/azuracast_backups/`
Of course you could use any other tool to do the same.
